gguf-py/gguf/constants.py

```diff
@@ -19,18 +19,59 @@
 
 class Keys:
     class General:
-        ARCHITECTURE         = "general.architecture"
-        QUANTIZATION_VERSION = "general.quantization_version"
-        ALIGNMENT            = "general.alignment"
-        NAME                 = "general.name"
-        AUTHOR               = "general.author"
-        VERSION              = "general.version"
-        URL                  = "general.url"
-        DESCRIPTION          = "general.description"
-        LICENSE              = "general.license"
-        SOURCE_URL           = "general.source.url"
-        SOURCE_HF_REPO       = "general.source.huggingface.repository"
-        FILE_TYPE            = "general.file_type"
+        ARCHITECTURE               = "general.architecture"
+        QUANTIZATION_VERSION       = "general.quantization_version"
+        ALIGNMENT                  = "general.alignment"
+        FILE_TYPE                  = "general.file_type"
+
+        # Authorship Metadata
+        NAME                       = "general.name"
+        AUTHOR                     = "general.author"
+        VERSION                    = "general.version"
+        ORGANIZATION               = "general.organization"
+
+        BASENAME                   = "general.basename"
+        FINETUNE                   = "general.finetune"
+        DESCRIPTION                = "general.description"
+        QUANTIZED_BY               = "general.quantized_by"
+        PARAMETER_CLASS_ATTRIBUTE  = "general.parameter_class_attribute"
+
+        # Licensing details
+        LICENSE                    = "general.license"
+        LICENSE_NAME               = "general.license.name"
+        LICENSE_LINK               = "general.license.link"
+
+        # Typically represents the converted GGUF repo (Unless native)
+        URL                        = "general.url"
+        DOI                        = "general.doi"
+        UUID                       = "general.uuid"
+        HF_URL                     = "general.huggingface.repository"
```

Comment:
Can we amend this and just do a regular repo_url? Keep it simple and flexible. This makes sense only in the context of the huggingface conversion script and nothing else. I understand what the spec says and I'm ignoring it for a reason. I'm not going to debate what the spec says or doesn't say because the implementation takes precedence as the spec is a WIP.